app/controllers/home_controller.rb

```diff
@@ -0,0 +1,19 @@
+class HomeController < ApplicationController
+
+  def top
+    @hands = Cards.new(nil)
+  end
+  def poker
+    @card = params[:card]
+    @hands = Cards.new(@card)
+    @hands.change
+    if @hands.valid? == true && @hands.uniqueness? == true
+      @hands.check(@hands.numbers,@hands.suits)
+      render("home/top")
+    else
+      @hands.error = "入力値が無効です"
```

Comment:
and don't create messages in controller in this case, please
please create them in service